src/components/card/card.scss

```diff
@@ -0,0 +1,15 @@
+
+	.card {
+		width: 400px;
+		height: 400px;
+		background: #ffffff;
+		border-radius: 5px;
+		margin: auto;
+		z-index: 1000;
+
+		@media screen and (max-width: 420px) {
+			width: 300px;
+		}
+	}
+
+	
```

Comment:
Fix the alignment here and add a new line at the end file

Reply:
okay @fortunee